v4/export/writer_test.go

```diff
@@ -117,7 +117,7 @@ func (s *testDumpSuite) TestWriteTableData(c *C) {
 	err = writer.WriteTableData(ctx, tableIR)
 	c.Assert(err, IsNil)
 
-	p := path.Join(dir, "test.employee.0.sql")
+	p := path.Join(dir, "test.employee.000000000.sql")
```

Comment:
Should there be a test that uses the 13 digit number?

Reply:
Added in https://github.com/pingcap/dumpling/pull/177/commits/ffcea135e9cbe26c577f2ea7004b3dc89f686044. The integration test in `rows` will also help check this.